.github/workflows/linux.yml

```diff
@@ -867,7 +867,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-8-cores' || 'ubuntu-20.04'}}
+    runs-on: ${{ github.event_name == 'schedule' && 'aks-linux-16-cores' || 'aks-linux-16-cores'}}
```

Comment:
Please specify on which `container` you will run, right now you use the same environment GitHub Runner is deployed and there's no guarantee what OS it will be (right now it is a container image based on Ubuntu 20.04, but it can be updated at any time because the purpose of it to just have the environment for GitHub Runner service)

Reply:
Done